src/manifest/clog.sidecar

```diff
@@ -10436,6 +10444,7 @@
         "MacroConfiguration": {
           "linux": "lttng_plus",
           "stubs": "stubs",
+          "macos": "STDOUT",
```

Comment:
I'm not convinced we practically want to do this.... the spew just for tests will be massive.